draft-ietf-quic-transport.md

```diff
@@ -3274,6 +3275,22 @@ The considerations for processing ICMP messages in the previous section also
 apply if these messages are used by DPLPMTUD.
 
 
+### PMTU Probes Containing Source Connection ID {#pmtu-probes-src-cid}
+
+Endpoints that rely on the destination connection ID for routing QUIC packets
+are likely to require that connection ID included in PMTU probe packets in order
+to route resulting ICMP messages ({{icmp-pmtud}}) back to the correct endpoints.
+
+Only long header packets ({{long-header}}) contain source connection IDs, but
+long header packets will not be acknowledged once the connection has been
```

Comment:
How about changing "will not be acknowledged once the connection has been established" to "will not be decrypted or acknowledged by the peer once the handshake is complete"?

I think that might help readers trying to understand the design.

Reply:
Added in the latest commit.